widget/window.go

```diff
@@ -223,7 +246,12 @@ func (w *Window) GetContainer() *Container {
 
 // Typically used internally.
 func (w *Window) SetCloseFunction(close RemoveWindowFunc) {
-	w.closeFunc = close
+	w.closeFunc = func() {
+		close()
+		w.MoveEvent.Fire(&WindowClosedEventArgs{
```

Comment:
This should be `ClosedEvent` instead if `MoveEvent`, right?

Reply:
Ah shit. The autocompletion won.
You're totally right.